ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -71,8 +71,8 @@ export default function GasTiming({
     ) {
       // getGasFeeTimeEstimate requires parameters in string format
       getGasFeeTimeEstimate(
-        new BigNumber(priority).toString(10),
-        new BigNumber(fee).toString(10),
+        new BigNumber(priority, 10).toString(10),
```

Comment:
Do you know if priority and fee are in base 10 when passed to BigNumber? Or are they base 16?

Reply:
It appears to be base 10, the value is coming from this hook https://github.com/MetaMask/metamask-extension/blob/develop/ui/hooks/useGasFeeInputs.js#L220 and is user input on advance gas form.

Reply:
And default is base 10, thus even previously it was base 10.

Reply:
I added some details here https://github.com/MetaMask/metamask-extension/pull/12140#issue-998970996, passing a base round-ups the value and saves from exception being thrown.